Documentation/Tutorials/edge-send-event-tutorial.md

```diff
@@ -0,0 +1,749 @@
+2. Code to accept opening the app via deep linking, and what to do with the incoming Assurance session ID (in our case, using it to initiate a connection with our Assurance session; examples of this can be found in the tutorial app)
+
+On the Assurance session side:
+1. The base URL to be set to the same value as the app URL (Assurance handles the rest of the connection link setup)
+
+> **Note**  
+> In Xcode the app URL can be configured using these steps:
+> 1. Select the project in the navigator.
+> 2. Select the app target in the `Targets` section, in the project configuration window.
+> 3. Select the `Info` tab.
+> 4. Set the desired deep linking URL.
+> ![Xcode deeplink app url config](../Assets/edge-send-event-tutorial/xcode-deeplink-app-url-config.jpg)
+> Please note that there is still code on the application side that is required for the app to respond to deep links; see the [guide on adding Assurance to your app](https://aep-sdks.gitbook.io/docs/foundation-extensions/adobe-experience-platform-assurance#add-the-aep-assurance-extension-to-your-app). For general implementation recommendations and best practices, see Apple's guide on [Defining a custom URL scheme for your app](https://developer.apple.com/documentation/xcode/defining-a-custom-url-scheme-for-your-app)
+
+</p></details>
+
+
+When presented with this window, your new Assurance session is ready to go, and it is now possible to connect the app to the session.  
+<img src="../Assets/edge-send-event-tutorial/assurance-create-session-qr.png" alt="Creating a new session in Assurance step 3 - QR code" width="400"/>
+<img src="../Assets/edge-send-event-tutorial/assurance-create-session-link.png" alt="Creating a new session in Assurance step 3 - Session link" width="400"/>
+
+### 2. Connect to the app   
+
+<details>
+  <summary> Details on connecting to Assurance </summary><p>
+
+There are two primary ways to connect an app instance to an Assurance session:
+1. QR Code: available with **Scan QR Code** option selected. Only works with physical devices, as it requires a physical device's camera to scan the code.
+2. Session Link: available with **Copy Link** option selected. Works with both physical and simulated devices.
+
+Note that both methods require setup on the app code side to allow for deep linking (see the section **What is a base URL?** under [Set up the Assurance session](#1-set-up-the-assurance-session)).
+
+To access these connection methods, click **Session Details** in the top right of the Assurance session page:  
+<img src="../Assets/edge-send-event-tutorial/assurance-session-details-qr.png" alt="Assurance Session Details - QR code" width="400"/>
+<img src="../Assets/edge-send-event-tutorial/assurance-session-details-link.png" alt="Assurance Session Details - Session link" width="400"/>
+
+You can edit both the **Session Name** and **Base URL**; changes to the **Base URL** value will automatically be reflected in both the QR code and session link.
+
+</p></details>
```

Comment:
This section seems informative, but seems unnecessary for the tutorial since you already go into details with the deeplink approach below. We can consider moving this somewhere in the Assurance docs instead.